lib/src/components/NxTable/NxTableBody.tsx

```diff
@@ -62,11 +64,24 @@ const NxTableBody = function NxTableBody(props: NxTableBodyProps) {
     </NxTableRow>
   );
 
+  function makeSkeletonChildren() {
+    const firstChild = React.Children.toArray(children)[0] as ReactElement;
+
+    return (
+      <>
```

Comment:
Are there any situations where a table would render less than 3 rows? In Kaleido I take it to imply that there won't be more than 3, but what are there possibilities of the table rendering 2 rows, and needing a 2row- skeleton loader? It may be how I'm understanding the Kaleido doc

Reply:
> Are there any situations where a table would render less than 3 rows? In Kaleido I take it to imply that there won't be more than 3, but what are there possibilities of the table rendering 2 rows, and needing a 2row- skeleton loader? It may be how I'm interpreting the Kaleido doc

The assumption is that a table on which skeleton-loading is being used would ultimately have a dynamic number of rows that wouldn't be known until the load completes.  So three was the chosen placeholder number.

Reply:
> > Are there any situations where a table would render less than 3 rows? In Kaleido I take it to imply that there won't be more than 3, but what are there possibilities of the table rendering 2 rows, and needing a 2row- skeleton loader? It may be how I'm interpreting the Kaleido doc
> 
> The assumption is that a table on which skeleton-loading is being used would ultimately have a dynamic number of rows that wouldn't be known until the load completes. So three was the chosen placeholder number.

Makes sense, thanks